code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -133,9 +133,16 @@
 	return ..()
 
 /**
- * Handler for `COMSIG_OBJ_HIDE`, connects only if `hide` is set to `TRUE`. Calls `update_cap_visuals` on pipe and its connected nodes
+ * Ran on `Initialize` if hide is `TRUE`, allows subtypes to register their own hiding logic.
+ * Including registering cap updating checks to run on different signals.
```

Comment:
Nit: 
Comments on procs like this should describe what it does/WHY it does things (to aid in understanding and prevent info rot)
Marking out where they're called risks drift, since that can change very easily without the person changing it actually noticing that the comment is wrong.

Reply:
right, right, makes sense